cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml

```diff
@@ -84,6 +84,8 @@ spec:
             - --container=heapster
             - --poll-period=300000
             - --estimator=exponential
+          securityContext:
+            runAsNonRoot: false
```

Comment:
They run as root, but they shouldn't. Setting runAsNonRoot sounded friendlier to the person who changes the user.